app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        self.__class__.alive.append(self)
```

Comment:
Use `self.alive.append(self)`

Reply:
Why? Alive is attribute of class

Reply:
better like this 
```
Animal.alive.append(self)
```

Reply:
Hi) it is hard coding((( If we change name of class we will have an error Better is self.__class__.alive.append(self) .IMHo